app/src/main/java/org/oppia/android/app/onboarding/OnboardingPagerAdapter.kt

```diff
@@ -2,65 +2,81 @@ package org.oppia.android.app.onboarding
 
 import android.content.Context
 import android.view.LayoutInflater
-import android.view.View
 import android.view.ViewGroup
-import android.widget.ScrollView
-import androidx.core.text.TextUtilsCompat
-import androidx.core.view.ViewCompat
-import androidx.viewpager.widget.PagerAdapter
+import androidx.recyclerview.widget.RecyclerView
 import org.oppia.android.databinding.OnboardingSlideBinding
 import org.oppia.android.databinding.OnboardingSlideFinalBinding
-import java.util.Locale
 
-/** Adapter to control the slide details in onboarding flow. */
+private const val VIEW_TYPE_TITLE_TEXT = 1
+private const val VIEW_TYPE_STORY_ITEM = 2
+
 class OnboardingPagerAdapter(
```

Comment:
Are we going to use the Bindale Adapter and remove this Adapter in this PR only or we are keeping this as a separate issue? @rt4914

Reply:
That seems like a larger & unrelated change--I suggest splitting it off of this task.

Reply:
Actually, given that so much of the adapter has changed anyway, maybe it is worth trying to migrate to BindableAdapter.

Reply:
@BenHenning So, should I replace it with a bindable Adapter in this PR only?

Reply:
Oh, I guess you are suggesting to track it via a new issue.

Reply:
Given the case that @prayutsu has already worked a lot on this issue, let him decide what he wants to do:
(a.) File an issue, mention it this PR and later on either @prayutsu solves it or the community.
(b.) Use `BindableAdapter` in this PR itself.